azure-pipelines.yml

```diff
@@ -453,8 +477,47 @@ stages:
             inputs:
               artifactName: conda_default
 
-          - task: GitHubRelease@1
+          - script: |
+              set -e
+              cd $(System.DefaultWorkingDirectory)
+              pip install flit
+              flit publish
+              echo "##vso[task.setvariable variable=pypi_published]True"
+            displayName: 'Publish to PyPi'
             condition: eq(variables['source_is_release_branch'], 'True')
+            env:
+              FLIT_PASSWORD: $(pypi_pw)
+              FLIT_USERNAME: $(pypi_user)
+
+          - script: |
+              set -e
+              cd $(System.DefaultWorkingDirectory)
+              eval "$(conda shell.bash hook)"
+              conda install anaconda-client
+              anaconda login --username "${CONDA_USERNAME}" --password "${CONDA_PASSWORD}"
+              anaconda upload --user BCG_Gamma $(System.ArtifactsDirectory)/conda_default/conda/noarch/gamma-pytools-*.tar.bz2
```

Comment:
You defined the `conda_default` as `artifactName` input on L478 - would it make sense to use that here?

Reply:
Not sure I understand you correctly – my intention: for the upload we are using the earlier built package, from the location where the DownloadBuildArtifact task earlier downloaded it too (I copied what was done for the GitHubRelease@1 task). I guess @rkdy can help

Reply:
Agree on the different order, will change it, thanks!